### DISCUSSION TITLE
Welcome to SEPP Discussions!

### DISCUSSION THREAD

**Discussion by geofmureithi** [geofmureithi/sepp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with payments and problems you face.

**Comment by paulmunyao** [comment]
Hello @geofmureithi
Great initiative you're taking.
My name is Paul Munyao Mutiso.
I would to have a platform that saves details of most commonly used numbers or payment information.
Then I wanted to know what tech stack will be used for the project as I use angular and Django, but I'm open to learning as well.
Thanks in advance Mureithi.

**Comment by paulmunyao** [comment]
@geofmureithi Another challenge that I do encounter when I make payment to the wrong person is that, I don't get the full amount. Case in point MPESA. When you make payment to a person who has fuliza and they receive the whole amount you do not get the full amount once refunded.

**Reply by geofmureithi** [reply]
Sadly the scope of the protocol would not cover such provider dependent cases. Nonetheless, we can work on preventing wrong payments.

**Comment by paulmunyao** [comment]
@geofmureithi Then don't forget the tech stack I will really appreciate

**Reply by geofmureithi** [reply]
The protocol is stack agnostic. Just like http you should be able to use any language.
Nonetheless I intend to make a compliant server using rust and maybe node. From there it should be easy to port it to other languages.

**Reply by geofmureithi** [reply]
Check out #2.
I hope it covers most of those issues.

**Reply by paulmunyao** [reply]
Yes it does.

**Comment by c12i** [comment]
Hey @geofmureithi curious to know what the progress on this project is? I am specifically interested in the specification. Have there have been strides made on this yet?

**Reply by c12i** [reply]
Also, I am assuming this would be an application layer protocol, built on top of TCP, is this correct?

**Reply by geofmureithi** [reply]
You are correct. I have been busy but will try and get this rolling over the holidays.

**Reply by c12i** [reply]
No worries man, will keep an eye on the repo
